src/indexPage/react-components/bot-landing/Carousel.jsx

```diff
@@ -1,88 +1,94 @@
 import React from 'react';
 import Slider from 'react-slick';
 import { translate } from '../../../common/i18n';
+// import arrow from '../../../../static/image/landing-carousel/arrow.svg'
 
 const carouselContentArray = [
     {
-        title      : translate('Where to find Binary Bot on Deriv'),
-        content    : translate('On the Deriv homepage, hit Trade at the top of the page and choose Binary Bot.'),
-        action_text: translate('Try Binary Bot on Deriv'),
-        img        : 'image/1a.webm',
+        action_text: translate('Check it out'),
+        img: 'image/landing-carousel/smart-trader.png',
     },
     {
-        title      : translate('It’s the same Binary Bot you know and love'),
-        content    : translate('Everything is exactly the way you like it.'),
-        action_text: translate('Try Binary Bot on Deriv'),
-        img        : 'image/2a.webm',
+        action_text: translate('Check it out'),
+        img: 'image/landing-carousel/binary-bot.png',
     },
     {
-        title      : translate('Bring your bots across seamlessly'),
-        content    : translate('Your strategies from Binary.com can be used on Deriv. Just load your XML files and get trading.'),
-        action_text: translate('Try Binary Bot on Deriv'),
-        img        : 'image/3a.webm',
+        action_text: translate('Learn more'),
+        img: 'image/landing-carousel/mt5.png',
+    },
+    {
+        action_text: translate('Learn more'),
+        img: 'image/landing-carousel/dtrader.png',
+    },
+    {
+        action_text: translate('Learn more'),
+        img: 'image/landing-carousel/dbot.png',
+    },
+    {
+        action_text: translate('Learn more'),
+        img: 'image/landing-carousel/deriv-go.png',
+    },
+    {
+        action_text: translate('Learn more'),
+        img: 'image/landing-carousel/derivX.png',
     },
 ]
 
 const Carousel = () => {
 
     const settings = {
-        dots          : false,
-        infinite      : true,
-        speed         : 1000,
-        slidesToShow  : 1,
-        slidesToScroll: 1,
-        arrows        : true,
-        autoplay      : true,
-        autoplaySpeed : 5000,
-        nextArrow     : <span className='arrow'><img src='./image/icon.svg' /></span>,
-        prevArrow     : <span className='arrow right'><img src='./image/icon.svg' /></span>,
-        responsive    : [
-            {
-                breakpoint: 1024,
-                settings  : {
-                    slidesToShow  : 1,
-                    slidesToScroll: 1,
-                },
-            },
-            {
-                breakpoint: 700,
-                settings  : {
-                    slidesToShow  : 1,
-                    slidesToScroll: 1,
-                },
-            },
-        ],
+        dots: false,
+        infinite: false,
+        speed: 300,
+        slidesToShow: 1,
```

Comment:
Can you change this slides to show to 5
and add this one as well, slidesToScroll: 1

Reply:
And also we need tie responsive breakpoints